client/app/locales/de.coffee

```diff
@@ -61,6 +61,7 @@ module.exports =
       "menu refresh initializing": "Initializing..."
       "menu refresh cleaning"   : "Cleaning..."
       "menu refresh indicator"  : "%{account}: %{mailbox} (%{progress}%)"
+      "menu last refresh"       : "Last refresh on %{date}."
```

Comment:
If I don't put something, the user will see "menu last refresh", so I'd rather put the proper english string into the german localization.